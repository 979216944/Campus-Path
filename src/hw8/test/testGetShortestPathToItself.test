# Verify that the program correctly gets shortest path 
# from a building to itself on campus map

# find shortest path from CSE to CSE
r
CSE
CSE

# quit
q