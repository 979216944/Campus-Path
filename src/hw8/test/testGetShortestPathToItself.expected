Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program correctly gets shortest path
# from a building to itself on campus map

# find shortest path from CSE to CSE
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Paul G. Allen Center for Computer Science & Engineering:
Total distance: 0 feet

Enter an option ('m' to see the menu):
# quit
